pkgs/desktops/gnustep/base/default.nix

```diff
@@ -35,6 +35,10 @@ gsmakeDerivation {
   ];
   patches = [ ./fixup-paths.patch ];
 
+  postPatch = ''
+    find -name '*.m' -exec sed -i 's|FALSE|false|g;s|TRUE|true|g' {} +
```

Comment:
Would you mind applying https://github.com/gnustep/libs-base/commit/06fa7792a51cb970e5d010a393cb88eb127830d7 instead?
```diff
diff --git i/pkgs/desktops/gnustep/base/default.nix w/pkgs/desktops/gnustep/base/default.nix
index 158971655b5..8c312d1faa2 100644
--- i/pkgs/desktops/gnustep/base/default.nix
+++ w/pkgs/desktops/gnustep/base/default.nix
@@ -1,7 +1,7 @@
 { aspell, audiofile
 , gsmakeDerivation
 , cups
-, fetchurl
+, fetchurl, fetchpatch
 , gmp, gnutls
 , libffi, binutils-unwrapped
 , libjpeg, libtiff, libpng, giflib, libungif
@@ -33,7 +33,13 @@ gsmakeDerivation {
     portaudio
     libiberty
   ];
-  patches = [ ./fixup-paths.patch ];
+  patches = [
+    ./fixup-paths.patch
+    (fetchpatch {
+      url = "https://github.com/gnustep/libs-base/commit/06fa7792a51cb970e5d010a393cb88eb127830d7.patch";
+      sha256 = "150n1sa34av9ywc04j36jvj7ic9x6pgr123rbn2mx5fj76q23852";
+    })
+  ];
```